GameServerLib/Game.cs

```diff
@@ -44,21 +42,20 @@ public class Game : IGame
         public float GameTime { get; private set; }
         private float _nextSyncTime = 10 * 1000;
 
-
-        public ObjectManager ObjectManager { get; private set; }
-        public Map Map { get; private set; }
-        public IPacketNotifier PacketNotifier { get; private set; }
-        public IPacketHandlerManager PacketHandlerManager { get; private set; }
+        private PacketServer _packetServer;
         public IPacketReader PacketReader { get; private set; }
+        public IPacketNotifier PacketNotifier { get; private set; }
+        public IObjectManager ObjectManager { get; private set; }
```

Comment:
Maybe merge ``PacketServer``, ``IPacketReader`` and ``IPacketNotifier`` into single interface called ``INetworkServer`` ?

Reply:
We need to remove all of these and replace it with API objects moving from and to the server lib, and the we will replace it with INetworkMessageHandler or something like that.